zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1232,9 +1315,40 @@ Both subsystems need to have sufficient amount of threads to achieve peak read t
     minute. This prevents herding during container deletion.
     Default is "10000".
 
+<a name="sc_adminserver_config"></a>
+
+#### AdminServer configuration
+
+**New in 3.5.0:** The following
+options are used to configure the [AdminServer](#sc_adminserver).
+
+* *admin.enableServer* :
```

Comment:
What is the status of SSL with the admin server? Should/could that not also be documented here?